packages/tasks/src/pipelines.ts

```diff
@@ -430,7 +430,7 @@ export const PIPELINE_DATA = {
 				name: "Image Captioning",
 			},
 		],
-		modality: "multimodal",
+		modality: "cv",
```

Comment:
Should this be "nlp"? The other labels seem to match the output modality.

Reply:
Hmm that would be a bit strange IMO. Both image generation and image captioning seem fundamentally computer vision tasks to me